blueos_repository/consolidate.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env python3
+import asyncio
+import dataclasses
+import json
+from pathlib import Path
+from typing import Any, AsyncIterable, Dict, List, Optional
+
+import aiohttp
```

Comment:
Potentially worth changing to httpx, per bluerobotics/BlueOS-docker#1054?
I don't have experience with either, so not sure what the differences would be.

Reply:
For the sake of this PR, that's heavily based over aiohttp, we can continue with it for now.